### ISSUE TITLE
bugsnag throws an error when dispatching custom click event in Electron webview

### ISSUE THREAD

**Issue by magne4000** [bugsnag/bugsnag-js#305]
### Expected behavior
Bugsnag should not crash when doing the following in Electron renderer console:
```javascript
document.querySelector('webview').dispatchEvent(new MouseEvent('click'));
```

### Observed behavior
```
TypeError client._logger.log is not a function 
    ...\node_modules\bugsnag-js\dist\bugsnag.js:1587:24 
```

### Steps to reproduce
Run the following snippet in an Electron renderer containing a webview:
```javascript
document.querySelector('webview').dispatchEvent(new MouseEvent('click'));
```

### Version
`4.1.2`

**Comment by alexstrat** [comment]
@bengourley can you please publish on `npm`. If not now, let us know in this thread when published. Thanks!

**Comment by bengourley** [comment]
Published `v4.1.3` 🚀
